### DISCUSSION TITLE
Nekropol

### DISCUSSION THREAD

**Discussion by k3746ov** [k3746ov/nekropol#1] [General]
![Screenshot_20220817-192836_Termux.jpg](https://user-images.githubusercontent.com/58249345/185181834-098efb45-eb4f-408b-a0b9-8fd4a6ae5b79.jpg)

[Telegram](https://t.me/linuxgnunix)

`fedreasonekropol`

<hr /><em>This discussion was created from the release <a href='https://github.com/k3746ov/NEKROPOL/releases/tag/android'>Nekropol</a>.</em>

**Comment by tiktox07-dev** [comment]
Hello anybody here
